en/reports/vivliostyle-report-2020/vf2020report.md

```diff
@@ -0,0 +1,261 @@
+---
+title: "Vivliostyle Foundation FY2020 Activity Report"
+lang: en
+---
+
+## **Chapter 1: Activity Report for FY2020**
+
+(The 3rd Fiscal Year: from April 1, 2020 to March 31, 2021)
+
+### 1. Organize developer/user events
+
+We have decided to hold developer/user events twice a year as a place to widely let users and developers know about Vivliostyle. This year, we were able to hold the following events.
+
+- [April 4, 2020: Vivliostyle User/Dev Meetup 2020 Spring (Held online)](https://vivliostyle.connpass.com/event/170939/)
+- [October 24, 2020: Vivliostyle User/Dev Meetup 2020 Autumn (Held online)](https://vivliostyle.connpass.com/event/189940/)
+
+The spring event coincided with the COVID 19 calamity. Initially, the event was scheduled to be held at the Japan Association of Graphic Arts Technology (JAGAT), but as Avoid the “Three Cs” was becoming a top priority, we were able to get through it by switching to online distribution. While most of the similar events were cancelled, thankfully, 142 people registered for the event. In terms of content, we were able to inform people about the real activities of our corporation, focusing on the report of Vivliostyle Pub, which was in the midst of alpha development at the time.
```

Comment:
online distribution? Do you mean you held the event online?

"calamity" is an odd choice here. Maybe "disaster" or just "pandemic"?